Source/WebKit/Platform/cocoa/LayerHostingContext.mm

```diff
@@ -170,6 +183,48 @@
 }
 
 #if USE(EXTENSIONKIT)
+#if ENABLE(MACH_PORT_LAYER_HOSTING)
+SafeSendRight LayerHostingContext::safeSendRight() const
+{
+    __block MachSendRight sendRight;
+    __block RetainPtr<NSData> dataRepresentation;
+    [[m_hostable handle] encodeWithBlock:^(mach_port_t copiedPort, NSData * _Nonnull data) {
+        sendRight = MachSendRight::create(copiedPort);
+        dataRepresentation = data;
+    }];
+    return { WTFMove(sendRight), FixedVector(makeVector(dataRepresentation.get())) };
```

Comment:
Seems inefficient to constructor a Vector, only to convert it to a FixedVector. I suspect we should use a span instead: `FixedVector { span(dataRepresentation.get()) }`, importing `wtf/cocoa/SpanCocoa.h`